x-pack/plugins/infra/public/components/metrics/index.tsx

```diff
@@ -0,0 +1,63 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License;
+ * you may not use this file except in compliance with the Elastic License.
+ */
+
+import { EuiLoadingChart, EuiPageContentBody, EuiTitle } from '@elastic/eui';
+import React from 'react';
+import styled from 'styled-components';
+import { InfraMetricData } from '../../../common/graphql/types';
+import { InfraMetricLayout, InfraMetricLayoutSection } from '../../pages/metrics/layouts/types';
+import { Section } from './section';
+
+interface Props {
+  metrics: InfraMetricData[];
+  layout: InfraMetricLayout[];
+  loading: boolean;
+}
+
+export class Metrics extends React.PureComponent<Props> {
+  public render() {
+    if (this.props.loading) {
+      return (
+        <LoadingContainer>
```

Comment:
I know we just created a loading component and I think that we should use  it to be consistent